docs/platforms/dotnet/index.mdx

```diff
@@ -38,35 +38,34 @@ Sentry captures data by using an SDK within your application's runtime. These ar
 
 Install the **NuGet** package to add the Sentry dependency:
 
-<OnboardingOption optionId="profiling">
+<OnboardingOption optionId="error-monitoring">
+
+  ```shell {tabTitle:.NET Core CLI}
+  dotnet add package Sentry -v {{@inject packages.version('sentry.dotnet') }}
+  ```
 
-  <Alert>
+  ```powershell {tabTitle:Package Manager}
+  Install-Package Sentry. -Version {{@inject packages.version('sentry.dotnet') }}
+  ```
 
-  Sentry profiling for .NET is available in Alpha on .NET 8.0 and newer.
+</OnboardingOption>
 
-  </Alert>
```

Comment:
Not sure I get what's going on here. The file name of this mdx file indicates it's for the default docs for dotnet right (nothing to do with profiling). The only place I see this alert in the current live docs is here though:
- https://docs.sentry.io/platforms/dotnet/profiling/

Presumably that's because of the `OnboardingOption optionId="profiling"` in the code. 

However, in [the preview](https://sentry-docs-git-fix-dotnet-onboarding-install.sentry.dev/platforms/dotnet/profiling/) for this PR, I'm still seeing this alert:
<img width="728" height="508" alt="image" src="https://github.com/user-attachments/assets/bb52e902-8a97-4ab9-8dc1-4fe69777fec1" />